aws/resource_aws_redshift_parameter_group_test.go

```diff
@@ -237,3 +281,33 @@ resource "aws_redshift_parameter_group" "bar" {
 }
 `, rInt)
 }
+
+func testAccAWSRedshiftParameterGroupConfigWithTags(rInt int, rString string) string {
+	return fmt.Sprintf(`
+resource "aws_redshift_parameter_group" "default" {
+  name        = "test-terraform-%[1]d"
+  family      = "redshift-1.0"
+  description = "Test parameter group for terraform"
+
+  tags = {
+    environment = "Production"
+	name     	= "test-terraform-%[1]d"
```

Comment:
It looks like the formatting here is off. Can you please adjust to 2 spaces and aligned like `terraform fmt`? Thanks.

Reply:
fixed!